Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -254,7 +254,7 @@ void GetVariable::execute_impl(Bytecode::Interpreter& interpreter) const
 
         auto reference_or_error = interpreter.vm().resolve_binding(string);
         if (reference_or_error.is_throw_completion()) {
-            interpreter.vm().throw_exception(interpreter.global_object(), reference_or_error.release_error().value());
+            interpreter.vm().throw_exception(interpreter.global_object(), *reference_or_error.release_error().value());
```

Comment:
Are these changes not meant for an earlier commit? (The change to `Optional<Value> value()`)

Reply:
Yep, fixed!